data/uking_functions.csv

```diff
@@ -410,7 +410,7 @@ Address,Quality,Size,Name
 0x000000710000ae74,U,000048,
 0x000000710000aea4,U,000040,
 0x000000710000aecc,U,000120,dragon::isWithin1kmOfHyruleCastle
-0x000000710000af44,U,000720,getDragonItemDropPosition
+0x000000710000af44,W,000720,_ZN5uking2ai25getDragonItemDropPositionEPN4sead7Vector3IfEERKS3_
```

Comment:
Should be marked as "m":

```suggestion
0x000000710000af44,m,000720,_ZN5uking2ai25getDragonItemDropPositionEPN4sead7Vector3IfEERKS3_
```

Also, given this function's location, I think it is probably part of the same translation unit as the file that contains the Dragon actor class, so perhaps this should be moved to Game/Actor/actDragon.cpp/h?